third_party/glad/generate.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
+
```

Comment:
I'm still unclear on why we need this wrapper file. Please add a file-level comment here explaining why we need a wrapper script.